### PR TITLE
Use asset panel

### PR DESCRIPTION
**Pull request by democrazyfi**

### CODE REVIEW

File: frontend/components/LspAssets.tsx [hunk 1]
```diff
@@ -102,49 +100,11 @@ export default function LspAssets() {
...
-        </Text>
-
-        {/* <ul>
-          {lps8Assets.map((asset, index) => (
-            <li key={index}>{asset}</li>
-          ))}
-        </ul> */}
+        ) :
+            lsp7Assets.map((asset, index) => (
+                    <LSP7Panel tokenName={asset.name!} tokenAmount={asset.balance!.toString()} tokenAddress={asset.address!} vaultAddress={graveVault!} />
```
Comment (Ledespana) [inline frontend/components/LspAssets.tsx:105]: they need `key`

Comment (Ledespana) [inline frontend/components/LspAssets.tsx:105]: We need `Processing` state in the Revive button, where click is disabled until both transactions are done.

File: frontend/components/LSP7Panel.tsx [hunk 2]
```diff
@@ -52,6 +49,55 @@ const LSP7Panel: React.FC<LSP7PanelProps> = ({
...
+        await signer.getAddress(),
+        tokenAmount,
+        false,
+        '0x',
+      ]);
+
+      const lsp9 = LSP9Vault__factory.connect(vaultAddress, provider);
+      await lsp9
+          .connect(signer)
+          .execute(0, tokenAddress, 0, lsp7Tx, { gasLimit: 400_00 });
```
Comment (Ledespana) [inline frontend/components/LSP7Panel.tsx:89]: We need a  Success Toast here

File: frontend/components/LspAssets.tsx [hunk 3]
```diff
@@ -102,49 +100,11 @@ export default function LspAssets() {
...
-          textAlign="center"
-        >
-          Coming soon!
-        </Text>
-
-        {/* <ul>
-          {lps8Assets.map((asset, index) => (
-            <li key={index}>{asset}</li>
-          ))}
-        </ul> */}
```
Comment (Ledespana) [inline frontend/components/LspAssets.tsx:147]: we still want this

**Review by Ledespana — APPROVED** [review]

File: frontend/components/LspAssets.tsx (hunk 3 above)
Comment (democrazyfi) [reply frontend/components/LspAssets.tsx:147]: We can add it once we have lsp8

**Pull request merged**